pom.xml

```diff
@@ -63,7 +63,7 @@
     </developers>
 
     <properties>
-        <druid.version>29.0.0</druid.version>
```

Comment:
I'd keep them separate, in case we'll need to add patches here

Reply:
Done